WriteHDLUtils.py

```diff
@@ -835,11 +934,19 @@ def writeModuleInst_generic(module, hls_src_dir, f_writeTemplatePars,
                     if not argname_is_array: break # We only need one match for non-arrays
     # end of loop
 
+    # External LUTs
```

Comment:
Why does this comment say "External LUTs"? I thought IR had been modified so all its LUTs are internal?

Reply:
With the LUTs that can be found in the LUTs/ directory moved inside the IR, there are still two ap_uints on the interface: https://github.com/cms-L1TK/firmware-hls/blob/IR_moveLUTs/TrackletAlgorithm/InputRouterTop.cc#L5-L6. (Maybe LUTs isn't the best description for the two numbers, but it was used in the IR so I went with that here as well) These bits are determined from the number of links etc, so I've just recreated the functions creating them in the python scripts. Sarah said that there might be a proper LUT created by the emulation for them in the future, but right now they're determined by functions in the IR testbench: https://github.com/cms-L1TK/firmware-hls/blob/IR_moveLUTs/TestBenches/InputRouter_test.cpp#L293